tests/BinEnvJsonTest.php

```
@@ -0,0 +1,339 @@
+
+use const JSON_PRETTY_PRINT;
+
+class BinEnvJsonTest extends TestCase
+{
+    private string $scriptPath;
+    private string $originalDir;
+    private string $tempDir;
+
+    protected function setUp(): void
+    {
+        $this->scriptPath = realpath(__DIR__ . '/../bin/envjson');
```

PHPStan (2.0.3)

Property Koriym\EnvJson\BinEnvJsonTest::$scriptPath (string) does not accept string|false.


(assign.propertyType)